src/_fade.scss

```diff
@@ -1,44 +1,65 @@
 /*
 * @author ivy
 * @module fade
-* @description anything to do with opacity/transparency
-* @note : this is just test code. feel free to change, I made it to illustrate 
-* how the documentation and project might work
+* @description manipulating opacity to fade in and out
 */
 
+
+/*
+	use keyframes fade for animation on page load
+*/
 @keyframes fade--in {
-    0% { opacity: 0; }
-    100% { opacity: 1; }
+	0% { opacity: 0; }
+	100% { opacity: 1; }
 }
 
 @keyframes fade--out {
-    0% { opacity: 1; }
-    100% { opacity: 0; }
+	0% { opacity: 1; }
+	100% { opacity: 0; }
 }
 
 .anim-fade--in {
-	animation: fade--in $duration-m ease-in;
-	animation-iteration-count: 1;
+	animation-delay:2s;
+	animation:fade--in $duration--long ease-in;
+	animation-iteration-count:1;
+	animation-fill-mode:forwards;
+}
+
+.anim-fade--in--short {
+	animation:fade--in $duration--short ease-in;
+	animation-iteration-count:1;
 	animation-fill-mode:forwards;
 }
 
 .anim-fade--out {
```

Comment:
ok - I'm trying to obey the PCV convention here maybe at the cost of legibility.
What I was wondering though is - should these be called fade/in/out or should the names of things be more consistent with the transitions so for example `anim-opacity-enter` and `anim-opacity-exit`? Its less catchy and fun then fade but...
naming is hard :|.
whats your thoughts?